src/lib/discord/Embed.ts

```diff
@@ -1,17 +1,8 @@
 import { ZeroWidthSpace } from '@utils/constants';
 import { MessageEmbed } from 'discord.js';
-import { Extendable, ExtendableStore } from 'klasa';
 
-export default class extends Extendable {
-	public constructor(store: ExtendableStore, file: string[], directory: string) {
-		super(store, file, directory, { appliesTo: [MessageEmbed] });
-	}
-
-	public addBlankField(this: MessageEmbed, inline?: boolean) {
-		return this.addField(ZeroWidthSpace, ZeroWidthSpace, inline ?? false);
-	}
-
-	public splitFields(this: MessageEmbed, contentOrTitle: string | string[], rawContent?: string | string[]) {
+export class Embed extends MessageEmbed {
```

Comment:
If we're not going to use Structures.extend for this then rename it to `SplitFieldsEmbed` to better represent what it adds

Reply:
We can't use Structures.extend for this.

Reply:
So rename it to `SplitFieldsEmbed` then